.github/workflows/ps8-build-release.yml

```diff
@@ -0,0 +1,222 @@
+name: PrestaShop 8 - Build & Release draft
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, edited, labeled]
+  push:
+    tags:
+      - "v8.*"
+    branches:
+      - "prestashop/8.x"
+
+env:
+  ZIP_NAME: ${{ github.event.repository.name }}-${{ github.ref_name }}
```

Comment:
Hello, I think that this env variable will override the one that is directly inside the jobs

Reply:
Discussed with you. Zip names for preprod and integration are OK.